scripts/run_integration_tests.sh

```diff
@@ -135,7 +135,7 @@ sudo setcap cap_net_admin+ep /usr/bin/qemu-system-x86_64
 sudo adduser $USER kvm
 newgrp kvm << EOF
 export RUST_BACKTRACE=1
-cargo test --features "integration_tests"
+cargo test --features "integration_tests" -- --nocapture
```

Comment:
May we move that one to its own PR? Or do we really need it here?

Reply:
I think the bigger issue is that we really need to redirect all console/serial output to a file and then print the full contents when the test completes (perhaps only do it on error)

Reply:
I can definitely move this one to another PR, and we can discuss there the details about console/serial redirection :)